generators/client/templates/angular/src/main/webapp/app/_polyfills.ts

```diff
@@ -1,3 +1,27 @@
 /* tslint:disable */
+// Polyfills
 import 'reflect-metadata/Reflect';
+// import 'core-js/es6';
+// Added parts of es6 which are necessary for your project or your browser support requirements.
+import 'core-js/es6/symbol';
+import 'core-js/es6/object';
+import 'core-js/es6/function';
+import 'core-js/es6/parse-int';
+import 'core-js/es6/parse-float';
+import 'core-js/es6/number';
+import 'core-js/es6/math';
+import 'core-js/es6/string';
+import 'core-js/es6/date';
+import 'core-js/es6/array';
+import 'core-js/es6/regexp';
+import 'core-js/es6/map';
+import 'core-js/es6/set';
+import 'core-js/es6/weak-map';
+import 'core-js/es6/weak-set';
+import 'core-js/es6/typed';
+import 'core-js/es6/reflect';
+
+import 'core-js/es7/reflect';
+
 import 'zone.js/dist/zone';
+import 'zone.js/dist/long-stack-trace-zone';
```

Comment:
Isn't this just for debugging? If this isn't directly linked to the ticket, let's remove this